libs/testserver/fake_workspace.go

```diff
@@ -135,6 +157,8 @@ func (s *FakeWorkspace) WorkspaceFilesExportFile(path string) []byte {
 }
```

Comment:
Should we also lock `WorkspaceFilesExportFile`?

Reply:
good catch, fixed.